service/history/workflow/retry.go

```diff
@@ -69,17 +75,18 @@ func getBackoffInterval(
 		return backoff.NoBackoff, enumspb.RETRY_STATE_NON_RETRYABLE_FAILURE
 	}
 
-	return nextBackoffInterval(now, currentAttempt, maxAttempts, initInterval, maxInterval, expirationTime, backoffCoefficient)
+	return NextBackoffInterval(now, currentAttempt, maxAttempts, initInterval, maxInterval, expirationTime, backoffCoefficient, ExponentialBackoffAlgorithm)
```

Comment:
This is only used within the current package, you don't need to export it publically by capitalizing it. Please keep this (and ExponentialBackoffAlgorithm) private unless we need to expose them